mikeio/__init__.py

```diff
@@ -121,11 +121,10 @@ def read(
     ext = Path(filename).suffix.lower()
 
     if "dfs" not in ext:
-        raise ValueError("mikeio.read() is only supported for Dfs files")
+        raise ValueError("mikeio.read() is only supported for dfs files")
 
     dfs = open(filename)
-    if isinstance(dfs, Mesh):
-        raise ValueError("mikeio.read() is not supported for Mesh files")
+    assert not isinstance(dfs, Mesh)
```

Comment:
I do not understand this change - isn't the user better supported by a ValueError with a clear error message?

Reply:
True